OpenOversight/app/templates/auth/user.html

```diff
@@ -1,19 +1,23 @@
 {% extends "base.html" %}
-{% import "bootstrap/wtf.html" as wtf %}
+{% import "bootstrap5/form.html" as wtf %}
 {% block title %}
   OpenOversight Admin - Update User
 {% endblock title %}
 {% block content %}
-  <div class="container theme-showcase" role="main">
+  <div class="container theme-showcase py-5" role="main">
     <div class="page-header">
       <h1>
         Update {{ user.username }}
         <small>{{ user.email }}</small>
       </h1>
     </div>
-    <div class="col-md-6 col-xs-12">
+    <div class="col-md-6 col-12">
```

Comment:
Replaced `col-xs` classes with `col-` to follow the change in Bootstrap 4
https://getbootstrap.com/docs/4.0/migration/#grid-system